src/utils.cpp

```diff
@@ -144,6 +144,24 @@ std::string ToCIFS(std::string& UNCName)
   return CIFSname;
 }
 
+bool InsertUser(std::string& UNCName)
+{
+  std::string SMBPrefix;
+  if (!g_szPass.empty())
+    SMBPrefix = "smb://" + g_szUser + ":" + g_szPass + "@";
+  else
+     SMBPrefix = "smb://" + g_szUser + "@";
+
+  size_t found = UNCName.find("smb://");
+  if (found != std::string::npos && found == 0)
```

Comment:
The `found != std::string::npos` check can be removed, checking for zero is enough.